.github/workflows/bump-sentry-in-getsentry.yml

```
@@ -0,0 +1,54 @@
+name: bump sentry in getsentry
+
+# Serializes the execution of this workflow.
+concurrency:
+  group: ${{ github.workflow }}
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  bump-sentry:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@d0651293c4a5a52e711f25b41b05b2212f385d28  # v3
+        with:
+          # For getsentry/bin/bump-sentry, sentry needs to be at ../sentry relative to getsentry.
+          path: sentry
+
+      - name: checkout getsentry
+        uses: actions/checkout@d0651293c4a5a52e711f25b41b05b2212f385d28  # v3
+        with:
+          repository: 'getsentry/getsentry'
+          path: getsentry
+          # This PAT (Personal Access Token) belongs to getsentry-bot,
+          # who can write to getsentry and is SAML+SSO ready.
+          token: ${{ secrets.BUMP_SENTRY_TOKEN }}
+
+      - name: install dependencies
+        shell: bash
+        run: |
+          cd getsentry
+          python -m pip install -q "$(grep '^pip-tools==' requirements-dev-frozen.txt)"
+
+      - name: bump-sentry ${{ inputs.sentry_sha }}
```

Is it still an input? won't this be this now
```suggestion
      - name: bump-sentry ${{ github.sha }}
```

---

Oh, yeah, forgot to change this part.